ingest/src/main/java/com/connexta/ingest/service/impl/IngestServiceImpl.java

```diff
@@ -28,13 +31,21 @@
 
   @NotNull private final S3StorageAdaptor s3Adaptor;
   @NotNull private final TransformClient transformClient;
+  private String callbackUrl;
 
   public IngestServiceImpl(
       @NotNull final S3StorageAdaptor s3Adaptor, @NotNull final TransformClient transformClient) {
     this.s3Adaptor = s3Adaptor;
     this.transformClient = transformClient;
   }
 
+  @Autowired
+  @Value("${endpointUrl.ingest.callback}")
+  public void setCallbackUrl(String callbackEndpoint) throws URISyntaxException {
+    LOGGER.info("Callback endpoint URL is: {}", callbackEndpoint);
+    this.callbackUrl = callbackEndpoint;
+  }
```

Comment:
✏️ Would it be safer to make `callbackUrl` and `transformEndpoint` final so that that are never null?